ui-tests/src/test/java/org/openmrs/reference/page/EditVisitNotePage.java

```diff
@@ -10,12 +10,24 @@
 package org.openmrs.reference.page;
 
 import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
 
 public class EditVisitNotePage extends VisitNotePage {
-
```

Comment:
Any reason for removing the existing empty line after `EditVisitNotePage` class declaration?

Reply:
To have a clean code description